test/headings.js

```diff
@@ -39,31 +39,31 @@ describe('headings', function () {
 
   it('allows a dash in generated DOM ids just like GitHub', function () {
     assert(~fixtures.github.indexOf('### heading with a - dash'))
-    $ = marky(fixtures.github)
+    $ = marky(fixtures.github, {prefixHeadingIds: false})
```

Comment:
I'm curious as to why these tests worked before; since they were lacking `{prefixHeadingIds: false}`, shouldn't they have been generating the `user-content-` prefix in the IDs? If not, then I might have something wrong in my logic and will need to amend my changes.